src/urllib3/exceptions.py

```diff
@@ -329,8 +312,6 @@ class InvalidChunkLength(HTTPError, httplib_IncompleteRead):
 
     partial: int
```

Comment:
Are these needed? We can add `self.partial: int = response.tell()` for example?

Reply:
You are right. Added!